classes/controller/Controller.php

```diff
@@ -684,7 +684,7 @@ protected function isCached($template, $cache_id = null, $compile_id = null)
      */
     public static function myErrorHandler($errno, $errstr, $errfile, $errline)
     {
-        if (error_reporting() === 0) {
+        if (!(error_reporting() & $errno)) {
```

Comment:
```suggestion
        /**
         * Prior to PHP 8.0.0, the $errno value was always 0 if the expression which caused the diagnostic was prepended by the @ error-control operator.
         * @see https://www.php.net/manual/fr/function.set-error-handler.php
         * @see https://www.php.net/manual/en/language.operators.errorcontrol.php
         *
         */
        if (!(error_reporting() & $errno)) {
```

Reply:
Ok for the change in 8.0 but is it retro-compatible with older versions? Should we perform a check and adapt this depending on the PHP version?